plugins/hydra_submitit_launcher/tests/test_submitit_launcher.py

```diff
@@ -33,51 +34,12 @@ class TestSubmititLauncher(LauncherTestSuite):
         (
             {
                 "defaults": [
-                    {"hydra/launcher": "submitit"},
+                    {"hydra/launcher": "submitit_local"},
                     {"hydra/hydra_logging": "hydra_debug"},
                     {"hydra/job_logging": "disabled"},
                 ],
                 "hydra": {
-                    "launcher": {
-                        "params": {
-                            "queue": "local",
-                            "folder": "${hydra.sweep.dir}/.${hydra.launcher.params.queue}",
-                            "queue_parameters": {
-                                "local": {
-                                    "gpus_per_node": 1,
-                                    "tasks_per_node": 1,
-                                    "timeout_min": 1,
-                                }
-                            },
-                        },
-                    }
-                },
-            },
-            ["-m"],
-        ),
-        # auto queue
-        (
-            {
-                "defaults": [
-                    {"hydra/launcher": "submitit"},
-                    {"hydra/hydra_logging": "hydra_debug"},
-                    {"hydra/job_logging": "disabled"},
-                ],
-                "hydra": {
-                    "launcher": {
-                        "params": {
-                            "queue": "auto",
-                            "folder": "${hydra.sweep.dir}/.${hydra.launcher.params.queue}",
-                            "queue_parameters": {
-                                "auto": {
-                                    "cluster": "local",
-                                    "gpus_per_node": 0,
-                                    "tasks_per_node": 1,
-                                    "timeout_min": 1,
-                                }
-                            },
-                        },
-                    }
+                    "launcher": {"params": {"gpus_per_node": 0, "timeout_min": 1}},
```

Comment:
actually this is not useful, that's just here to prove that it works

Reply:
how is this proving it works? I don't think it's validating that the configs are actually making it all the way in.

Reply:
it's not validating anything, it basically proves that hydra works and accepts the parameters, nothing more :D
I like keeping this around so I can try with other overrides easily if need be